test/testcafe/spec/NumberChallengePushView_spec.js

```diff
@@ -0,0 +1,83 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import ChallengeOktaVerifyPushPageObject from '../framework/page-objects/ChallengeOktaVerifyPushPageObject';
+
+import numberChallenge from '../../../playground/mocks/data/idp/idx/authenticator-verification-number-challenge';
+import success from '../../../playground/mocks/data/idp/idx/success';
+
+const logger = RequestLogger(/challenge|challenge\/poll/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+const numberChallengeSuccessMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(numberChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(success);
+
+const numberChallengeWaitMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(numberChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
+  .respond(numberChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/resend')
+  .respond(numberChallenge);
+
+fixture('Challenge Okta Verify Push');
```

Comment:
Can you add a test that stop poll once session expires or in case of any errors?